ci/pipeline

```diff
@@ -75,13 +76,14 @@ def uploadTarballToS3(): Option[awsClient.Artifact] = utils.stage("Upload Packag
 /**
  * Packages Marathon and uploads its artifacts alongside sha1 checksum to S3.
  *
+ * @param upload Indicates whether we want to upload the tarball package.
  * @return Version and artifact description of Marathon build.
  */
-def createAndUploadPackages(): (String, Option[awsClient.Artifact]) = {
+def createAndUploadPackages(upload: Boolean = true): (String, Option[awsClient.Artifact]) = {
```

Comment:
This grew and got out of hand. We should refactor here a little. Lets do 2 methods:
- `createTarballPackage()`
- `uploadArtifatct(artifact: Artifact)`

Reply:
And we should put all creation of packages (Tarball, Docker, Linux) into one stage.